src/gamma_inc.jl

```diff
@@ -421,39 +421,35 @@ See also: [`gamma_inc(a,x,ind)`](@ref SpecialFunctions.gamma_inc)
 """
 function gamma_inc_taylor(a::Float64, x::Float64, ind::Integer)
     acc = acc0[ind + 1]
-    wk = zeros(30)
-    flag = false
-    apn = a + 1.0
-    t = x/apn
-    wk[1] = t
-    loop = 2
-    for indx = 2:20
-       apn += 1.0
-       t *= x/apn
-       if t <= 1.0e-3
-           loop = indx
-           flag = true
-           break
-       end
-       wk[indx] = t
-    end
-    if !flag
-        loop = 20
+    wk = zeros(20)
+    apn = a
+
+    # compute and store larger terms in wk, to add from small to large
+    t = 1
```

Comment:
Both `t` and `sm` are type unstable.

Reply:
:+1:  Only `t`, it seems, and only in the first iteration though it seems? An easy fix would be
```suggestion
    t = 1.0
```

Reply:
I think fixing just `t` will fix the whole issue 😄 and ya just in the first iteration!
```julia
julia> @code_warntype gamma_inc_asym(5.5, 1.2, 0)
MethodInstance for gamma_inc_asym(::Float64, ::Float64, ::Int64)
  from gamma_inc_asym(a::Float64, x::Float64, ind::Integer) in Main at REPL[2]:1
Arguments
  #self#::Core.Const(gamma_inc_asym)
  a::Float64
  x::Float64
  ind::Int64
Locals
  @_5::Union{Nothing, Tuple{Float64, Tuple{Base.OneTo{Int64}, Int64}}}
  q::Float64
  sm::Union{Float64, Int64}
  i::Int64
  t::Union{Float64, Int64}
  amn::Float64
  wk::Vector{Float64}
  acc::Any
...
```